#global testerror "Err0042"
func hello(name: string) {
    return "hello" ++ name
}
